### ISSUE TITLE
Text in Placeholder text shouldn't be typeable

### ISSUE THREAD

**Issue by katelovescode** [pieforproviders/pieforproviders#1986]
## 👉🏼 Steps to Reproduce

1. Click on "Record New"
2. Click on "Payment"
3. Check a box that says "I received a different amount from the state"
4. Click inside the "Enter Amount" words and hit backspace once or twice
5. The text is editable

## 🇺🇸 States this bug is happening in
- [ ] ALL
- [X] Nebraska
- [ ] Illinois

## 🧥Expected behavior

Instead of being editable, when someone clicks inside the box the placeholder text of "Enter Amount" should disappear.

## 📸 Screenshots

![typable](https://user-images.githubusercontent.com/8364647/144272275-16ac85e7-5c82-44dc-80b4-ed8dc7ea15bc.gif)

## 💻 Browser Info

- Operating System: Mac Monterey
- Browser: Chrome
- Version: 96.0.4664.55 (Official Build) (x86_64)

**Comment by katelovescode** [comment]
This is weird - I'm not sure if I'm linking this right, CodePen has changed a lot since I used it last, but check this out: https://codepen.io/katelovescode/pen/xxXoLOR?editors=001 with a placeholder that _isn't_ editable.  I just added a placeholder attribute to the core code demo of Ant Design, so I'm super confused why it's editable in our code?  I thought maybe the formatter might be impacting it so I tried that too: https://codepen.io/katelovescode/pen/JjrQyRE?editors=001 placeholder is on the first input.  @cjhaddad any thoughts?
